enterprise/frontend/src/metabase-enterprise/api/content-translation.ts

```diff
@@ -0,0 +1,54 @@
+import { invalidateTags, listTag } from "metabase/api/tags";
+import type { DictionaryArray } from "metabase/i18n/types";
```

Comment:
if this is a type that is sent to or returned from the api it should be in `metabase-types/api`